### PR TITLE
Add Accumulator TVL

### PR DESCRIPTION
**Pull request by PotluckProtocol**
##### Name (to be shown on DefiLlama): 
Accumulator

##### Twitter Link: 
https://twitter.com/accu_defi

##### Website Link: 
https://accumulator.finance

##### Logo (High resolution, will be shown with rounded borders):
https://www.dropbox.com/scl/fi/m1iu628qnb8vspmh91jac/monogram.png?rlkey=s46je8zy7h43xisypz11b5r9w&dl=0

##### Current TVL:
$172,447

##### Chain:
Shimmer(EVM)

##### Short Description (to be shown on DefiLlama):
Accumulator is a yield optimizer utilizing various strategies to increase LP positions.

##### Category (full list at https://defillama.com/categories) *Please choose only one:
Yield Aggregator

##### forkedFrom (Does your project originate from another project):
Beefy

##### methodology (what is being counted as tvl, how is tvl being calculated):
LP deposited into our strategy vaults.

##### Github org/user (Optional, if your code is open source, we can track activity):

### CODE REVIEW

**Comment by llamatester** [comment]
Error while running adapter at projects/accumulator: 
        
 
  
    
    
    Error: 
        Unknown chain(s): shimmer
        Note: if you think that the chain is correct but missing from our list, please add it to 'projects/helper/chains.json' file

**Comment by llamatester** [comment]
The adapter at projects/accumulator exports TVL: 
        
 
 
    shimmer_evm               170.68 k
    
    total                    170.68 k

File: projects/accumulator/index.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+const utils = require("../helper/utils");
+const { toUSDTBalances } = require("../helper/balances");
+let _response;
+
+function fetchChain(chainId) {
+  return async () => {
+    if (!_response)
+      _response = utils.fetchURL("https://newapi.potluckprotocol.com/tvl");
```
Comment (g1nt0ki) [inline projects/accumulator/index.js:8]: we prefer computing tvl on chain, is there some api to get vault list?

Comment (PotluckProtocol) [reply projects/accumulator/index.js:8]: Yes, its beefy API fork, and here is the list of our shimmer pools.

https://raw.githubusercontent.com/PotluckProtocol/potluck-vaults/main/src/features/configure/vault/shimmer_pools.js

**Comment by llamatester** [comment]
The adapter at projects/accumulator exports TVL: 
        
 
 
    shimmer_evm               218.90 k
    
    total                    218.90 k

**Pull request merged**